ms/MSOper/MSMetaData.cc

```diff
@@ -3399,6 +3399,8 @@ void MSMetaData::_computeScanAndSubScanProperties(
     uInt chunkSize = nrows/nchunks;
     if (nrows % nchunks > 0) {
         ++chunkSize;
+        // integer division
+        nchunks = nrows/chunkSize + 1;
     }
```

Comment:
easy mistake to make with the nchunks++, doesn't work due to the maximum number of 1000 chunks. Good the testsuite catches that.
One could also write it like this:

```
uInt chunkSize = max(1u, nrows / 1000u);
uInt nchunks = nrows / chunkSize;
if (nrows % chunkSize != 0) {
    nchunks++;
}
```